src/components/DayTab.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="day-tab">
+    <button class="day-tab__button"><slot /></button>
```

Comment:
Is it necessary to have this `div` that wraps a `button`? If so, than you need to remember that e.g. `click` event should be handled on a `button` and not on a root element.